.github/workflows/build_with_conda.yml

```diff
@@ -1,6 +1,14 @@
 name: Build with conda
 on:
   push:
+    branches:
+      - master
+  # For Pull-Requests, this runs the CI on merge commit
+  # of HEAD with the target branch instead on HEAD, allowing
```

Comment:
typo, `on HEAD` should be `of HEAD`

I've read the docs. I'm still not seeing the difference though? Previously it'd run on push, now it'll run on PR update AND push for master. What's the difference?

Reply:
In the previous setup, the CI was run on the tip of the branch (referred to as `HEAD`) rather than on the result of the merge. Hence you could and up with a passing CI on the PR but a failing on `master` after the merge.

This setup makes sure that the CI on PR are run on the merge commit; hence if the CI is green on Pull Request it also will on `master` after the merge (except failures due to network latency or to CI provider services').

Reply:
On, that's objectively better. Okay, understood - thanks for the explanation.